### PR TITLE
Add rsa backend

### PR DESCRIPTION
**Pull request by sosthene-nitrokey**
This patch adds the RSA backend that will be necessary for the PIV application and for the updated OpenPGP smartcard.
The rsa backend is added behind the `rsa` feature flag, which is currently not enabled for anything.

### CODE REVIEW

File: components/apps/Cargo.toml [hunk 1]
```diff
@@ -28,10 +29,11 @@ default = [
     "fido-authenticator",
     "ndef-app",
     "oath-authenticator",
-    "trussed/clients-3"
+    "trussed/clients-3",
 ]
 alpha = ["opcard", "trussed/clients-4"]
 provisioner = ["provisioner-app", "trussed/clients-3"]
+rsa = ["trussed-rsa-alloc"]
```
Comment (robin-nitrokey) [inline components/apps/Cargo.toml:36]: For consistency with the auth backend, please rename the feature to `backend-rsa` and move it to the `# backends` section.

Comment (sosthene-nitrokey) [reply components/apps/Cargo.toml]: Done

Comment (robin-nitrokey) [reply components/apps/Cargo.toml]: `src/dispatch.rs` still uses `rsa`.

File: 0001-Fix-builder-for-podman.patch [hunk 2]
```diff
@@ -0,0 +1,104 @@
+From c7cff70975b2ab6d82966dea3fa7b8fe4883d6ca Mon Sep 17 00:00:00 2001
+From: =?UTF-8?q?Sosth=C3=A8ne=20Gu=C3=A9don?= <sosthene@nitrokey.com>
+Date: Wed, 15 Feb 2023 09:21:40 +0100
+Subject: [PATCH] Fix builder for podman
```
Comment (robin-nitrokey) [inline 0001-Fix-builder-for-podman.patch:4]: Accidentally added to the PR?

Comment (sosthene-nitrokey) [reply 0001-Fix-builder-for-podman.patch]: :facepalm:

File: components/apps/Cargo.toml (hunk 1 above)
Comment (sosthene-nitrokey) [reply components/apps/Cargo.toml]: :facepalm:

**Review by robin-nitrokey — APPROVED** [review]

**Pull request merged**